pkg/compatibility/three_to_four_test.go

```diff
@@ -0,0 +1,264 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compatibility
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/gcp-service-broker/brokerapi/brokers/models"
+	"github.com/GoogleCloudPlatform/gcp-service-broker/db_service"
+	"github.com/jinzhu/gorm"
+	"github.com/pivotal-cf/brokerapi"
+
+	// Import so the brokers register with the registry
+	_ "github.com/GoogleCloudPlatform/gcp-service-broker/brokerapi/brokers"
+)
+
+// These plan details are dumped from a 3.6 service broker to be realistic, the
+// list also includes user-defined plans that were saved to the database.
+const planDetails = `INSERT INTO plan_details (id, created_at, updated_at, deleted_at, service_id, name, features) VALUES
+  ('000ad064-f354-4f2d-a615-672fc2b98551','2018-08-22 19:16:49','2018-08-22 19:16:49',NULL,'b8e19880-ac58-42ef-b033-f7cd9c94d1fe','bigtable-ssd-20','{\"num_nodes\":\"20\",\"storage_type\":\"SDD\"}'),
```

Comment:
It appears these GUIDs are being used inline throughout a few places in the code. I wonder if it might be worth while to define them as constants for consistency's sake.

Reply:
What do you think about pulling them out within the compatibility package for any locations they're standalone strings (e.g. in this database table dump they'd stay hard-coded for readability)?

Reply:
SGTM. Thanks.